source/blood/src/menu.cpp

```diff
@@ -1559,6 +1571,15 @@ void SetWeaponSwitch(CGameMenuItemZCycle *pItem)
 
 extern bool gStartNewGame;
 
+void ShowDifficulties()
+{
+    gGameMenuMgr.Push(&menuDifficulty, 3);
```

Comment:
Seems to be too hackish

Reply:
Yeah, like the whole game engine and everything.

Reply:
Pushed new commit regarding this. Maybe I could reduce the hackishness a little.